File: C only/Class Problems/wpassign-12,9/q3.c
#include <stdio.h>

int main()
{
	int n;
	printf("Enter the number of elements :- ");
	scanf("%d",&n);
	int array[n];
	printf("Enter the Elements :- ");
	for(int f=0; f < n; f++) {
		scanf("%d",&array[f]);
	}
	int target;
	printf("Enter the Target Sum :- ");
	scanf("%d",&target);
	for(int i=0;i<n ; i++){
	    for(int j=i+1;j<n;j++){
	        if(array[i] + array[j] == target){
	            printf("(%d,%d) ",array[i],array[j]);
	        }
	    }
	}
}

File: JavaScript/Assign2/q3.js
var array = [1,3,4,5,6,9,6];
var length = array.length;
var sum =0,avg;
for(i =0;i<length;i++){
    sum = sum + array[i];
}
avg = sum / length;
console.log("The Average of all elements =",avg);